seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/config/JdbcOptions.java

```diff
@@ -84,7 +84,7 @@ public interface JdbcOptions {
     Option<Integer> FETCH_SIZE =
             Options.key("fetch_size")
                     .intType()
-                    .defaultValue(0)
+                    .defaultValue(1024)
```

Comment:
Changing the default value may cause loss of optimization logic inside the driver. It is better to keep it as 0 and let the driver decide the default value itself.